packages/pluto/readme.md

```diff
@@ -0,0 +1,59 @@
+# Pluto
+
+### An end to end testing framework.
+
+## Setting up for local development
+
+### Clone this repository
+
+```shell
+git clone git@github.com/creately/modules-js.git
+cd modules-js/packages/pluto
+npm install
```

Comment:
It should be possible to install pluto with.

```
npm install -g @creately/pluto
```

Reply:
Noted, thanks.